benchmarks/linear_model.py

```diff
@@ -80,25 +100,74 @@ def peakmem_fit(self, *args):
         ridge.fit(self.X, self.y)
 
 
+class Linear_bench(Benchmark):
+    """
+    Benchmarks for Linear Reagression.
+    """
+
+    # params = (representation)
+    param_names = ["params"] + Benchmark.param_names
+    params = ([("dense",), ("sparse",)],) + Benchmark.params
+
+    def setup(self, params, *common):
+        representation = params[0]
+        self.n_jobs = common[0]
+
+        self.X, self.y = _synth_regression_dataset(5000, 100000,
+                                                   representation)
+
+    def time_fit(self, *args):
+        linear_reg = LinearRegression(n_jobs=self.n_jobs)
+        linear_reg.fit(self.X, self.y)
+
+    def peakmem_fit(self, *args):
+        linear_reg = LinearRegression(n_jobs=self.n_jobs)
+        linear_reg.fit(self.X, self.y)
+
+
+class SGDRegressor_bench(Benchmark):
+    # params = (representation)
+    param_names = ["params"]
+    params = ([("dense",), ("sparse",)],)
+
+    def setup(self, params):
+        representation = params[0]
+
+        self.X, self.y = _synth_regression_dataset(5000, 100000,
+                                                   representation)
+
+    def time_fit(self, *args):
+        sgd_reg = SGDRegressor(max_iter=2000, tol=1e-16)
+        sgd_reg.fit(self.X, self.y)
+
+    def peakmem_fit(self, *args):
+        sgd_reg = SGDRegressor(max_iter=2000, tol=1e-16)
+        sgd_reg.fit(self.X, self.y)
+
+
 class ElasticNet_bench(Benchmark):
     """
     Benchmarks for ElasticNet.
     """
+
     # params = (representation, precompute)
-    param_names = ['params']
-    params = ([('dense', True),
-               ('dense', False),
-               ('sparse', True),
-               ('sparse', False)],)
+    param_names = ["params"]
+    params = (
+        [
+            ("dense", True),
+            ("dense", False),
+            ("sparse", True),
+            ("sparse", False),
+        ],
+    )
```

Comment:
why are you changing formatting like this? The original is PEP8 compliant, more compact and more readable (for me at least). There are lots of changes like this in this PR which makes it harder to review.

Reply:
I'm using an auto-formatter and I guess it is default settings. I reverted most of these changes.